src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs

```diff
@@ -403,6 +403,7 @@ private byte[] ConvertToBytes(object inputObject)
                     if (_lastInputType != null && baseType != _lastInputType)
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
FormatHex.InputObject From Pipeline.can process jagged input: { $true, $false, $true, 123, 100, 76, $true, $false }
```powershell
Expected regular expression '0000000000000000   01 00 00 00 00 00 00 00 01 00 00 00              ? ?' to case sensitively match 0000000000000000   01 00 00 00 00 00 00 00 01 00 00 00              ?       ?       , but it did not match.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Format-Hex.Tests.ps1: line 336
336:                 $Results[$Number] | Should -MatchExactly $ExpectedResults[$Number]
```